src/dashboard/components/DonationSummary.vue

```diff
@@ -136,10 +165,17 @@ export default {
       });
     },
     distribute() {
+      const store_dist = this.$store.state.metamask.distribution;
       this.distribution = this.creators.map(c => {
```

Comment:
The component distribution contains creator info while the store distribution is just a `creator id -> share` mapping since that is the only relevant information in the following steps of the checkout process where it's (the store distribution) is intended to be used.

Reply:
After considering changes made in #208, I think this was a poor design decision. Will amend in the future.